File: DevExpress-Demo/CS/MapDemo.Wpf/Helpers/ChartControlService.cs
using DevExpress.Mvvm.UI;
using DevExpress.Xpf.Charts;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MapDemo {
    public interface IChartControlService {
        void Animate();
        void UpdateData();
    }
    public class ChartControlService : ServiceBase, IChartControlService {
        public ChartControl ChartControl { get { return (ChartControl)AssociatedObject; } }
        public void UpdateData() { 
            ChartControl.UpdateData();
        }
        public void Animate() {
            ChartControl.Animate();
        }
    }
}

File: DevExpress-Demo/VB/MapDemo.Wpf/Helpers/ChartControlService.vb
Imports DevExpress.Mvvm.UI
Imports DevExpress.Xpf.Charts

Namespace MapDemo

    Public Interface IChartControlService

        Sub Animate()

        Sub UpdateData()

    End Interface

    Public Class ChartControlService
        Inherits ServiceBase
        Implements IChartControlService

        Public ReadOnly Property ChartControl As ChartControl
            Get
                Return CType(AssociatedObject, ChartControl)
            End Get
        End Property

        Public Sub UpdateData() Implements IChartControlService.UpdateData
            ChartControl.UpdateData()
        End Sub

        Public Sub Animate() Implements IChartControlService.Animate
            ChartControl.Animate()
        End Sub
    End Class
End Namespace
